### PR TITLE
Added undergraduate lab guide page. Other fixes too.

### PR DESCRIPTION
**Pull request by PiersonGarrett**
Edited the kinetic installation guide so that some more dependencies will be resolved. Added new page for undergraduate-guide.

### CODE REVIEW

File: _posts/wiki/2019-09-03-Undergrad-Guide.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
+---
+title: Getting Started as an Undergraduate in the HIRO Group
+description: Answering the questions you didn't know to ask
+permalink: join_us.html
```
Comment (alecive) [inline _posts/wiki/2019-09-03-Undergrad-Guide.md]: you should change the permalink to `undergrad_guide.html`
Also, you should add yourself as an author!

File: _posts/wiki/2019-09-03-Undergrad-Guide.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+---
+
+# Contents
+This guide is intended for new undergraduate members with limited exposure to
```
Comment (alecive) [inline _posts/wiki/2019-09-03-Undergrad-Guide.md]: Although I'm fine with calling this "undergraduate guide", I'd wager we should limit the use of the "undergraduate", given that this applies generally to "new members with limited exposure to ROS and ..."

File: _posts/wiki/2019-09-03-Undergrad-Guide.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+ROS and other tools used in the HIRO Group. This page in conjunction with the
+new member to-do list page will help you through setting up all of the software
```
Comment (alecive) [inline _posts/wiki/2019-09-03-Undergrad-Guide.md]: There are ways to reference other pages within Jekyll, you should do it here! See line 16 of https://github.com/HIRO-group/HIRO-group.github.io/blob/master/_posts/wiki/2018-06-14-ROS-indigo-installation.md for an example

File: _posts/wiki/2019-09-03-Undergrad-Guide.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+1. **Google it!**
+Take the error messages or type what you are trying to do and search for answers
+using Google. This should always be your first step after trying for a solid
+15-20 minutes on your own. Carefully read through at least the first page for
+answers to your problems and refer to the tutorials. Try this for an hour, then
+if you can not figure it out, ask one of the other undergraduates.
+
+2. **Ask an Undergraduate Student!**
+You should introduce yourself to everyone in the lab and get to know them
+(everyone is also listed on the website under people). Just like for any other problem,
```
Comment (alecive) [inline _posts/wiki/2019-09-03-Undergrad-Guide.md]: Also here, please refer to the webpage directly by using markdown

**Comment by alecive** [comment]
Adding also @joewie and @Sayter99 if they want to add/modify something.

**Comment by joewie** [comment]
> Adding also @joewie and @Sayter99 if they want to add/modify something.

I already looked through it before Garrett made the PR, and I think it's great!

**Comment by alecive** [comment]
Sounds great! I'll merge it now.

**Pull request merged**